krun.py

```diff
@@ -132,7 +132,11 @@ def create_arg_parser():
                         help=("Enable developer mode"))
     parser.add_argument("--info", action="store_true",
                         help=("Print session info for specified "
-                              "config file and exit")),
+                              "config file and exit"))
+    parser.add_argument("--strip-results", action="store",
+                        metavar="keyspec",
```

Comment:
`metavar="KEYSPEC",` is the usual convention (used below in `metavar="FILENAME",`).